holidays/countries/gambia.py

```diff
@@ -0,0 +1,195 @@
+            self._add_observed(dt)
+
+        # Prophet's Birthday.
+        for dt in self._add_mawlid_day(tr("Mawlid Nabi")):
+            self._add_observed(dt)
+
+        # Laylat al-Qadr.
+        for dt in self._add_laylat_al_qadr_day(tr("Lialat-Ul-Qadr")):
+            self._add_observed(dt)
+
+        # Eid al-Fitr.
+        name = tr("Koriteh")
+        self._add_eid_al_fitr_day(name)
+        if self._year >= 2021:
+            for dt in self._add_eid_al_fitr_day_two(name):
+                self._add_observed(dt)
+
+        # Eid al-Adha.
+        name = tr("Tobaski")
+        self._add_eid_al_adha_day(name)
+        if self._year >= 2021:
+            for dt in self._add_eid_al_adha_day_two(name):
+                self._add_observed(dt)
+
+
+class GM(Gambia):
+    pass
+
+
+class GMB(Gambia):
+    pass
+
+
+class GambiaIslamicHolidays(_CustomIslamicHolidays):
+    # https://web.archive.org/web/20240716225449/https://www.timeanddate.com/holidays/gambia/ashura
+    ASHURA_DATES = {
+        2015: (OCT, 24),
+        2016: (OCT, 12),
+        2017: (OCT, 1),
+        2018: (SEP, 20),
+        2019: (SEP, 10),
+        2020: (AUG, 29),
+        2021: (AUG, 19),
+        2022: (AUG, 8),
+        2023: (JUL, 28),
+        2024: (JUL, 16),
+        2025: (JUL, 5),
+    }
+
+    # https://web.archive.org/web/20241209170416/https://www.timeanddate.com/holidays/gambia/eid-al-adha
+    EID_AL_ADHA_DATES = {
+        2015: (SEP, 24),
+        2016: (SEP, 13),
+        2017: (SEP, 2),
+        2018: (AUG, 22),
+        2019: (AUG, 11),
+        2020: (JUL, 31),
+        2021: (JUL, 20),
+        2022: (JUL, 9),
+        2023: (JUN, 28),
+        2024: (JUN, 16),
+        2025: (JUN, 6),
+    }
+
+    # https://web.archive.org/web/20241204114244/https://www.timeanddate.com/holidays/gambia/eid-al-fitr
+    EID_AL_FITR_DATES = {
+        2015: (JUL, 18),
+        2016: (JUL, 7),
+        2017: (JUN, 26),
+        2018: (JUN, 15),
+        2019: (JUN, 4),
+        2020: (MAY, 23),
+        2021: (MAY, 13),
+        2022: (MAY, 2),
+        2023: (APR, 21),
+        2024: (APR, 10),
+        2025: (MAR, 30),
+    }
+
+    # https://web.archive.org/web/20240909111449/https://www.timeanddate.com/holidays/gambia/prophet-birthday
+    MAWLID_DATES = {
+        2016: (DEC, 12),
+        2017: (DEC, 1),
+        2018: (NOV, 21),
+        2019: (NOV, 9),
+        2020: (OCT, 29),
+        2021: (OCT, 19),
+        2022: (OCT, 8),
+        2023: (SEP, 28),
+        2024: (SEP, 16),
+    }
+
+    # https://web.archive.org/web/20241209070551/https://www.timeanddate.com/holidays/gambia/laylat-al-qadr
+    LAYLAT_AL_QADR_DATES = {
+        2015: (JUL, 14),
+        2016: (JUL, 3),
+        2017: (JUN, 22),
+        2018: (JUN, 11),
+        2019: (JUN, 1),
+        2020: (MAY, 20),
+        2021: (MAY, 9),
+        2022: (APR, 29),
+        2023: (APR, 18),
+        2024: (APR, 6),
+        2025: (MAR, 27),
+    }
```

Comment:
```suggestion
    LAYLAT_AL_QADR_DATES_CONFIRMED_YEARS = (2015, 2025)
    LAYLAT_AL_QADR_DATES = {
        2016: (JUL, 3),
        2022: (APR, 29),
    }
```
And this should be placed before Mawlid (alphabetical order).